Token.js

```diff
@@ -40,18 +40,13 @@ const availableToAoe = [
 
 
 const debounceLightChecks = mydebounce(() => {		
-		if(window.walls != undefined){
-			if(window.walls.length < 5){
-				redraw_light_walls();	
-			}
-		}
-		else{
+		if(window.walls?.length < 5){
 			redraw_light_walls();	
 		}
 		redraw_light();
 		if(!window.DM)
 			check_token_visibility();
-}, 100);
+}, 250);
```

Comment:
increased debounce to make it feel smoother while person is moving  quickly. Thoughts? I thought about changing this depending on image size? shorten for smaller images since processing is quicker?

Reply:
If it feels smoother it's good with me. I just had it in there for when moving multiple tokens as once it didn't get called for all of them dropping. And I had to worry less about doubling up on it.